src/components/screens/forms/form-history-screen.tsx

```diff
@@ -1,12 +1,198 @@
+        return (
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField sx={{ fontWeight: "bold" }}>{ params.colDef.headerName }</AdminFormTypographyField>
+          </AdminFormListStack>
+        );
+      },
+      renderCell: params => {
+        const formattedDate = moment(params.row.createdAt).format("DD.MM.YYYY HH:mm:ss");
+        return (
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField>{ formattedDate }</AdminFormTypographyField>
+          </AdminFormListStack>
+        );
+      }
+    },
+    {
+      field: "userId",
+      headerName: strings.formHistoryScreen.historyTable.actor,
+      width: 400,
+      renderHeader: params => {
+        return (
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField sx={{ fontWeight: "bold" }}>{ params.colDef.headerName }</AdminFormTypographyField>
+          </AdminFormListStack>
+        );
+      },
+      renderCell: params => {
+        const user = metaformMembers.find(member => member.id === params.row.userId);
+        const userName = user === undefined ? strings.generic.unknown : `${user.firstName} ${user.lastName}`;
+        return (
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField>{ userName }</AdminFormTypographyField>
+          </AdminFormListStack>
+        );
+      }
+    },
+    {
+      field: "replyId",
+      headerName: strings.formHistoryScreen.historyTable.replyId,
+      width: 400,
+      renderHeader: params => {
+        return (
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField sx={{ fontWeight: "bold" }}>{ params.colDef.headerName }</AdminFormTypographyField>
+          </AdminFormListStack>
+        );
+      },
+      renderCell: params => {
+        return (
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField>{ params.row.replyId }</AdminFormTypographyField>
+          </AdminFormListStack>
+        );
+      }
+    },
+    {
+      field: "logEntryType",
+      headerName: strings.formHistoryScreen.historyTable.actionPerformed,
+      width: 400,
+      renderHeader: params => {
+        return (
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField sx={{ fontWeight: "bold" }}>{ params.colDef.headerName }</AdminFormTypographyField>
+          </AdminFormListStack>
+        );
+      },
+      renderCell: params => {
+        return (
+          <AdminFormListStack direction="row">
+            <AdminFormTypographyField>{ LocalizationUtils.getLocalizedAuditLogEntryType(params.row.logEntryType) }</AdminFormTypographyField>
+          </AdminFormListStack>
+        );
+      }
+    }
+  ];
+
+  /**
+   * Component render
+   */
+  return (
+    <>
+      <NavigationTabContainer>
+        <NavigationTab
+          text={ strings.navigationHeader.formsScreens.formScreen }
+          to="answers"
+        />
```

Comment:
Clicking on this appends _answers_ to the current path e.g. it navigates to `/admin/forms/{slug}/history/answers` while it should navigate to `/admin/forms/{slug}/answers`